Tests/SmokeDynamoDBTests/DynamoDBCompositePrimaryKeyTableUpdateItemConditionallyAtKeyTests.swift

```diff
@@ -684,20 +638,19 @@ class DynamoDBCompositePrimaryKeyTableUpdateItemConditionallyAtKeyTests: XCTestC
             XCTFail("Unexpected error thrown: \(error).")
         }
         
-        let secondRetrievedItem: TestTypeADatabaseItem? = try! table.getItem(forKey: key).wait()
+        let secondRetrievedItem: TestTypeADatabaseItem? = try await table.getItem(forKey: key)
         
         XCTAssertNil(secondRetrievedItem)
     }
     
-    func testUpdateItemConditionallyAtKeyWithUnknownItemWithAsyncProvider() {
-        let table = InMemoryDynamoDBCompositePrimaryKeyTable(eventLoop: eventLoop)
+    func testUpdateItemConditionallyAtKeyWithUnknownItemWithAsyncProvider() async throws {
```

Comment:
This test is now identical to `testUpdateItemConditionallyAtKeyWithUnknownItem`.

Reply:
Removed